============================= test session starts ==============================
platform darwin -- Python 3.12.0, pytest-7.4.4, pluggy-1.4.0
rootdir: /private/var/folders/z0/zvd81kyj08xgnt8c1hbb2f780000gn/T/pytest-of-whenessel/pytest-3/test_broken_fixture0
plugins: allure-pytest-2.13.4.dev2+gd759bc5
collected 1 item

test_broken_fixture.py E                                                 [100%]

==================================== ERRORS ====================================
________________ ERROR at setup of test_broken_fixture_example _________________

    @pytest.fixture
    def broken_fixture():
>       raise IndexError
E       IndexError

test_broken_fixture.py:5: IndexError
=============================== warnings summary ===============================
../../../../../../../../../Users/whenessel/Development/PycharmProjects/allure-python/.venv/lib/python3.12/site-packages/_pytest/config/__init__.py:735
  /Users/whenessel/Development/PycharmProjects/allure-python/.venv/lib/python3.12/site-packages/_pytest/config/__init__.py:735: PytestAssertRewriteWarning: Module already imported so cannot be rewritten: allure_pytest
    self.import_plugin(arg, consider_entry_points=True)

-- Docs: https://docs.pytest.org/en/stable/how-to/capture-warnings.html
=========================== short test summary info ============================
ERROR test_broken_fixture.py::test_broken_fixture_example - IndexError
========================= 1 warning, 1 error in 0.01s ==========================
